napalm/base/base.py

```diff
@@ -28,8 +28,49 @@
 from napalm.base.exceptions import ConnectionException
 
 
+yang_filters = {
+    "interfaces": {
+        "openconfig": {"include": ["/openconfig-interfaces:interfaces"], "exclude": []},
+        "ntc": {"include": [], "exclude": []},
+    },
+    "vlans": {
+        "openconfig": {
+            "include": [
+                "/openconfig-network-instance:network-instances/network-instance/name",
+                "/openconfig-network-instance:network-instances/network-instance/config",
+                "/openconfig-network-instance:network-instances/network-instance/vlans",
+            ],
+            "exclude": [],
+        },
+        "ntc": {"include": ["/ntc-vlan:vlan/config/vlans"], "exclude": []},
+    },
+}
+
+
+def get_yang_filter(path, model):
+    try:
+        include = yang_filters[path][model]["include"]
+        exclude = yang_filters[path][model]["exclude"]
+    except KeyError:
+        raise NotImplementedError("{}: {} not supported".format(path, model))
+
+    if not include and not exclude:
+        raise NotImplementedError("{}: {} not supported".format(path, model))
+
+    return include, exclude
+
+
 class NetworkDriver(object):
-    def __init__(self, hostname, username, password, timeout=60, optional_args=None):
+    def __init__(
+        self,
+        hostname,
+        username,
+        password,
+        rosetta_driver=None,
+        yang_model="openconfig",
```

Comment:
I'm thinking maybe there are other ways to select across different standardization orgs within the same session, i.e., without re-initializing the driver? I think that'd add a pretty good degree of flexibility.

Reply:
Maybe, for the getters and setters it's going to be easier to have a default model associated to it but as an enhancement we could add a parameter to them to override the default one or have a way to override it, something like `device.set_yang_model("something")`.  In any case, I'd suggest treating that as a post enhancement to avoid complicating this initial PR.